gui/default/syncthing/core/syncthingController.js

```diff
@@ -682,6 +693,29 @@ angular.module('syncthing.core')
             });
         };
 
+        $scope.dismissAuthenticationSetupSuggestion = function () {
+            try {
+                window.localStorage["authWarningDismissed"] = true;
+                var expire = 7*24*60*60*1000;  // Miliseconds (7 days * 24 hours * 60 mins * 60 secs * 1000 miliseconds)
```

Comment:
I most definitely don't want to get that banner every week on my multi-user systems with only one user - I wouldn't expire at all.

Reply:
I agree that such reminder can be annoying. So should I rewrite my PR in way that this notification will be using `config.options.unackedNotificationIDs` and will work as others?

Reply:
In my opinion yes.